assignments/function-conversion.js

```diff
@@ -1,23 +1,44 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+myFunction = (() =>{
+
+});
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
-
+anotherFunction= ((param) =>{
+    return param;
+});
```

Comment:
✅ 
you can also write it as this `let anotherFunction = (param) => param;`
By doing it in this one liner, we don't need the curly braces `{}` or the `return` keyword, as this syntax implicitly knows to return.